src/sentry/api/endpoints/project_ownership.py

```diff
@@ -24,7 +25,27 @@ def validate_raw(self, attrs, source):
                 u'Parse error: %r (line %d, column %d)' % (
                     e.expr.name, e.line(), e.column()
                 ))
-        attrs['schema'] = dump_schema(rules)
+
+        schema = dump_schema(rules)
+
+        bad_actors = []
+        for rule in rules:
+            for owner in rule.owners:
+                try:
+                    resolve_actor(owner, self.context['ownership'].project_id)
```

Comment:
We will definitely want to create a bulk version of this in the near-ish term. It would be useful elsewhere too.